src/utils/auth.js

```diff
@@ -0,0 +1,8 @@
+export const getJwToken = async () => {
+  const response = localStorage.getItem('token')
```

Comment:
I think we can just return `localStorage.getItem('token')` without the variable assignment.